smart_contracts.rst

```diff
@@ -566,7 +566,9 @@ Called after a channel has been closed. Can be called by any Ethereum address an
 
     Only a valid signed :term:`balance proof` from the channel's ``closing participant`` (the other channel participant) ``MUST`` be accepted. This :term:`balance proof` sets the amount of tokens owed to the ``non-closing participant`` by the ``closing participant``.
 
-    Only a valid signed balance proof update message  ``MUST`` be accepted. This message is a confirmation from the ``non-closing participant`` that the contained :term:`balance proof` can be set on his behalf.
+    Only a valid signed `balance proof update`__ message  ``MUST`` be accepted. This message is a confirmation from the ``non-closing participant`` that the contained :term:`balance proof` can be set on his behalf.
+
```

Comment:
On the same note, the word `message` might be redundant here in the 2 places it occurs. Consider removing it.

Reply:
I see the word "message" in many similar sentences in this document.  I'll consider changing them together in a new issue https://github.com/raiden-network/spec/issues/144.